src/libstd/collections/hash/map.rs

```diff
@@ -2447,9 +2447,12 @@ impl RandomState {
         // every corresponding HashMap a different iteration order.
         thread_local!(static KEYS: Cell<[u64; 2]> = {
             let mut buf = [0u8; 16];
-            // in case of `Error::UNAVAILABLE` we use a constant value
+            // In case of `Error::UNAVAILABLE` we use a constant value
+            // for the following whitelisted targets
+            let whitelisted = cfg!(target="wasm32-unknown-unknown");
             match getrandom::getrandom(&mut buf) {
-                Ok(()) | Err(Error::UNAVAILABLE) => (),
+                Ok(()) => (),
```

Comment:
Is it `=> {}` preferred?

Reply:
Hm, `std` has both variants, but looks like `=> {}` is more popular. Will change it in the next commit.